hierarc/LensPosterior/ddt_kin_gauss_constraints.py

```diff
@@ -64,11 +66,17 @@ def __init__(
         :param hernquist_approx: bool, if True, uses the Hernquist approximation for the light profile
         :param kappa_ext: mean of the external convergence from which the ddt constraints are coming from
         :param kappa_ext_sigma: 1-sigma distribution uncertainty from which the ddt constraints are coming from
+        :param kappa_ext_alt: mean of the alternative external convergence from which the ddt constraints are coming from
```

Comment:
I don't fully understand the meaning of these parameters. Can you further elaborate on this?

Reply:
So similarly to kappa_ext and kappa_ext_sigma, kappa_ext_alt and kappa_ext_alt_sigma describes a population distribution for kappa_ext.  However, it describes an alternative population, sampled for flagged systems (see kappa_alt_population parameter later).  So a way I plan to use it is kappa_ext and kappa_ext_sigma describes the population kappa_ext distribution for the SL2S sample, and kappa_ext_alt and kappa_ext_alt_sigma describes the population kappa_ext distribution for the SLACS sample.